File: Module.Persist.TPM/Model/TPM/AssortmentMatrix.cs
using Core.Data;
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Module.Persist.TPM.Model.TPM
{
    public class AssortmentMatrix : IEntity<Guid>, IDeactivatable
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid Id { get; set; }

        public bool Disabled { get; set; }

        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
        public int Number { get; set; }
        public DateTimeOffset? DeletedDate { get; set; }
        public DateTimeOffset? StartDate { get; set; }
        public DateTimeOffset? EndDate { get; set; }
        public DateTimeOffset? CreateDate { get; set; }


        public int ClientTreeId { get; set; }
        public Guid ProductId { get; set; }

        public virtual Product Product { get; set; }
        public virtual ClientTree ClientTree { get; set; }

        public virtual AssortmentMatrix2Plu Plu { get; set; }


    }
}

File: Module.Frontend.TPM/Client/model/tpm/assortmentmatrix/AssortmentMatrix.js
Ext.define('App.model.tpm.assortmentmatrix.AssortmentMatrix', {
    extend: 'Ext.data.Model',
    idProperty: 'Id',
    breezeEntityType: 'AssortmentMatrix',
    fields: [
        { name: 'Id', hidden: true },
        { name: 'Number', type: 'int', hidden: false, isDefault: true, isKey: true },
        { name: 'ClientTreeId', hidden: true, isDefault: true },
        {
            name: 'ClientTreeName', type: 'string', mapping: 'ClientTree.Name',
            defaultFilterConfig: { valueField: 'Name' }, breezeEntityType: 'ClientTree', tree: true, hidden: false, isDefault: true
        },
        {
            name: 'PluCode', type: 'string', mapping: 'Plu.PluCode',
            defaultFilterConfig: { valueField: 'PluCode' }, breezeEntityType: 'Plu', hidden: false, isDefault: true
        },
        { name: 'ProductId', hidden: true, isDefault: true },
        {
            name: 'ProductEAN_PC', type: 'string', mapping: 'Product.EAN_PC', 
            defaultFilterConfig: { valueField: 'EAN_PC' }, breezeEntityType: 'Product', hidden: false, isDefault: true
        },
        { name: 'StartDate', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: 'EndDate', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        { name: 'CreateDate', type: 'date', hidden: false, isDefault: true, timeZone: +3, convert: dateConvertTimeZone },
        {
            name: 'ClientTreeObjectId', type: 'int', mapping: 'ClientTree.ObjectId',
            defaultFilterConfig: { valueField: 'ObjectId' }, hidden: false, isDefault: true,            
        }
    ],
    proxy: {
        type: 'breeze',
        resourceName: 'AssortmentMatrices',
        reader: {
            type: 'json',
            totalProperty: 'inlineCount',
            root: 'results'
        },
        extraParams: {
            needActualAssortmentMatrix: false
        }
    }
});
